pythran/toolchain.py

```diff
@@ -391,7 +419,7 @@ def compile_pythrancode(module_name, pythrancode, specs=None,
 
 
 def compile_pythranfile(file_path, output_file=None, module_name=None,
-                        cpponly=False, **kwargs):
+                        cpponly=False, pyonly=False, **kwargs):
```

Comment:
seeing that change, I see a pattern emerging :-) I'll formalize that and issue a PR against your PR!